glotaran/builtin/megacomplexes/spectral/shape.py

```diff
@@ -95,7 +59,17 @@ def calculate_gaussian(self, axis: np.ndarray) -> np.ndarray:
             -np.log(2) * np.square(2 * (axis - self.location) / self.width)
         )
 
-    def calculate_skewed_gaussian(self, axis: np.ndarray) -> np.ndarray:
+
+@model_item(
+    properties={
+        "skewness": Parameter,
+    },
+    has_type=True,
+)
+class SpectralShapeSkewedGaussian(SpectralShapeGaussian):
+    """A skewed Gaussian spectral shape"""
+
+    def calculate(self, axis: np.ndarray) -> np.ndarray:
         r"""Calculate the skewed Gaussian shape for ``axis``.
```

Comment:
The docstring needs to be updated in line 111 to point at `SpectralShapeGaussian.calculate` instead of the old `calculate_gaussian`.

Reply:
Fixed.